image_width: 752
image_height: 480
camera_name: cam3
camera_matrix:
  rows: 3
  cols: 3
  data: [464.69998552912386, 0, 371.54871319955964, 0, 464.7198988074454, 265.49508717929507, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.29623784881669385, 0.0904035194005803, -0.000748313563150221, 0.000537999914149228, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [464.69998552912386, 0, 371.54871319955964, 0, 0, 464.7198988074454, 265.49508717929507, 0, 0, 0, 1, 0]

